labelbox/data/serialization/ndjson/classification.py

```diff
@@ -1,3 +1,4 @@
+from tkinter import N
```

Comment:
**Suggestion** maybe this was added via your editor? if so, let's drop

Reply:
yes